includes/class-wc-payload-gateway.php

```diff
@@ -281,6 +283,9 @@ public function create_token( $payment_method ) {
 		$token->set_user_id( get_current_user_id() );
 		$token->save();
 
+		$pm = new Payload\PaymentMethod( array( 'id' => $payment_method['id'] ) );
+		$pm->update( array( 'attrs' => array( '_wp_token_id' => $token->get_id() ) ) );
+
```

Comment:
I don't know PHP but would it be better formatting to have this on several lines?